lib/Service/ConfigService.php

```diff
@@ -348,15 +377,7 @@ public function sendPasswordByMail() {
 	 *
 	 */
 	public function enforcePasswordProtection(DeprecatedCircle $circle) {
-		if ($this->isContactsBackend()) {
-			return false;
-		}
-
-		if ($circle->getSetting('password_enforcement') === 'true') {
-			return true;
-		}
-
-		return ($this->config->getAppValue('sharebymail', 'enforcePasswordProtection', 'no') === 'yes');
+		return false;
```

Comment:
Same here.

Reply:
same, deprecated. Those methods are called by some old files that needs to be cleaned. Should be another PR